src/client/client.ts

```diff
@@ -157,6 +157,15 @@ export type PublishJsonRequest = Omit<PublishRequest, "body"> & {
 
 export type EventsRequest = {
```

Comment:
I think this would provide a better interface

```
export type EventsRequest = {
  cursor?: number;
  filter?: EventsRequestFilter
};

type EventsRequestFilter = {
  messageId?: string;
  state?: State;
  url?: string;
  topicName?: string;
  scheduleId?: string;
  queueName?: string;
  fromDate?: number; // unix timestamp (ms)
  toDate?: number; // unix timestamp (ms)
  count?: number;
}
```

what do you think?

Reply:
Yeah I like that